docs/docs/advanced-topics/3-indices.md

```diff
@@ -146,10 +146,10 @@ For more information see the [Postgres documentation](https://www.postgresql.org
 
 ## Disabling synchronization
 
-TypeORM does not support some index options and definitions (e.g. `lower`, `pg_trgm`) due to many database-specific differences and multiple
-issues with getting information about existing database indices and synchronizing them automatically. In such cases you should create the index manually
-(for example, in migrations) with any index signature you want. To make TypeORM ignore these indices during synchronization, use `synchronize: false`
-option on the `@Index` decorator.
+TypeORM does not support some index options and definitions (e.g. `lower`, `pg_trgm`) because of lot of different database specifics and multiple
+issues with getting information about exist database indices and synchronizing them automatically. In such cases you should create index manually
+(for example in the [migrations](../migrations/01-why.md)) with any index signature you want. To make TypeORM ignore these indices during synchronization use `synchronize: false`
+option on `@Index` decorator.
```

Comment:
I find the prior version more clear, and some of the changes here are incorrect.

What's the goal for the changes to this paragraph?